tgui/packages/fulpui-patches/interfaces/AntagInfoBloodsucker.tsx

```diff
@@ -105,22 +152,114 @@ const BloodsuckerIntro = () => {
   );
 };
 
-export const AntagInfoBloodsucker = (props, context) => {
-  const [tab, setTab] = useLocalState(context, 'tab', 1);
+const BloodsuckerClan = (props: any, context: any) => {
+  const { act, data } = useBackend<BloodsuckerInformation>(context);
+  const { clan, in_clan } = data;
+
+  if (!in_clan) {
+    return (
+      <Section minHeight="220px">
+        <Box mt={5} bold textAlign="center" fontSize="40px">
+          You are not in a Clan.
+        </Box>
+        <Box mt={3}>
+          <Button
+            fluid
+            icon="users"
+            content="Join Clan"
+            textAlign="center"
+            fontSize="30px"
+            lineHeight={2}
+            onClick={() => act('join_clan')}
+          />
+        </Box>
+      </Section>
+    );
+  }
+
   return (
-    <Window width={620} height={580}>
-      <Window.Content>
-        <Tabs>
-          <Tabs.Tab
-            icon="list"
-            lineHeight="23px"
-            selected={tab === 1}
-            onClick={() => setTab(1)}>
-            Introduction
-          </Tabs.Tab>
-        </Tabs>
-        {tab === 1 && <BloodsuckerIntro />}
-      </Window.Content>
-    </Window>
+    <Stack vertical fill>
+      <Stack.Item minHeight="20rem">
```

Comment:
A single item stack seems a bit odd.

Reply:
I would remove the excess stack and have the Section as the root component

Reply:
When I do this, going to this section of the UI causes the whole UI to scroll down, making the tabs unreachable.